docs/index.md

```diff
@@ -1,23 +1,23 @@
 # On becoming an OBO Semantic Engineer
 
-Welcome the OBOOK, the OBO Organized Knowledge portal and our OBO Semantic Engineering Training!
+Welcome to the OBOOK and our OBO Semantic Engineering Training!
 
-## Introduction to the OBOOK 
+## Introduction to the OBOOK (Open Biological and Biomedical Ontologies Organized Knowledge)
 
-Documentation in the OBOOK is organised based on the [Diataxis framework](https://diataxis.fr/) of documentations into 4 distinction sections: 
+Documentation in the OBOOK is organised into 4 distinct sections based on the [Diátaxis framework](https://diataxis.fr/) of documentation: 
```

Comment:
@matentzn is á allowed in this form?

Reply:
I think its fine. Just double check it later when looking at the deployed site.